connector/src/main/scala/com/microsoft/kusto/spark/datasource/KustoOptions.scala

```diff
@@ -55,6 +55,13 @@ object KustoOptions {
   // An integer number corresponding to the period in seconds after which the operation will timeout. Default: '5400' (90 minutes)
   val KUSTO_TIMEOUT_LIMIT: String = newOption("timeoutLimit")
 
+  // A json representation of the ClientRequestProperties object used for reading from Kusto
+  var KUSTO_CLIENT_REQUEST_PROPERTIES_JSON: String = newOption("clientRequestPropertiesJson")
+  // A json representation of the SparkIngestionProperties object used for writing to Kusto
+  var KUSTO_SPARK_INGESTION_PROPERTIES_JSON: String = newOption("sparkIngestionPropertiesJson")
```

Comment:
That is not the palace for ingestion specifics.
Let's add KustoIngestionOption file, that will inherit KustoOptions, and contain the relevant ingestion options.